File: src/main/java/com/yong/kakaobot/request/MessageRequest.java
package com.yong.kakaobot.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class MessageRequest {
	@JsonProperty("user_key")
	private String userKey;
	private Type type;
	private String content;

	enum Type{
		text, photo
	}
}

File: src/main/java/com/yong/kakaobot/request/MessageRequest.kt
package com.yong.kakaobot.request

import com.fasterxml.jackson.annotation.JsonProperty

data class MessageRequest(
        @JsonProperty("user_key")
        var userKey:String? = null,
        var type:Type? = null,
        var content:String? = null){

    enum class Type{
        text, photo
    }
}